packages/core/admin/admin/src/components/Layouts/HeaderLayout.tsx

```diff
@@ -35,7 +35,7 @@ const BaseHeaderLayout = React.forwardRef<HTMLDivElement, BaseHeaderLayoutProps>
           background="neutral0"
           shadow="tableShadow"
           width={`${width}px`}
-          zIndex={1}
+          zIndex={10}
```

Comment:
Hello @Manoj-M-S Thanks a lot for your contribution.
I would like to ask you to fix the issue not in the Header (because it is used in different places in the repo and this modification can cause other errors) but instead work on the zIndex of the Accordion, for example, in the Api tokens you can modify the value here https://github.com/strapi/strapi/blob/fc9d26d995624dc886b29f563e1de655d47e6609/packages/core/admin/admin/src/pages/Settings/pages/ApiTokens/EditView/components/CollapsableContentType.tsx#L74
Thanks a lot